made fields final

simarsingh24 opened the pull request:
Fixes #2437

iamareebjamal commented on android/app/src/main/java/org/fossasia/openevent/core/discount/DiscountFragmentViewModel.java:
```diff
@@ -20,9 +20,9 @@
+    private final RealmDataRepository realmRepo;
     private MutableLiveData<Boolean> discountCodesDownloadResponse;
-    private CompositeDisposable compositeDisposable;
+    private final CompositeDisposable compositeDisposable;
```
Pull them together on top

simarsingh24 replied:
okay

iamareebjamal reviewed (changes requested):
Put final fields on top and add a line feed between them

iamareebjamal commented on android/app/src/main/java/org/fossasia/openevent/core/sponsor/SponsorsFragmentViewModel.java:
```diff
@@ -12,9 +12,9 @@
     private LiveData<List<Sponsor>> sponsorsList;
-    private RealmDataRepository realmRepo;
-
+    
```
extra

iamareebjamal reviewed (changes requested):
Separate final and non-final fields by a line feed

iamareebjamal reviewed (changes requested):
Please check previous request for changes

iamareebjamal commented:
@simarsingh24 Please resolve the conflicts

simarsingh24 committed:
made fields final


minor changes


minor changes


minor change


removed extra line feed


added line feed after final fields


minor changes

simarsingh24 commented:
resolve conflicts
